src/Network/Network/NetworkManager/NetworkManagerSecurityAdminConfiguration/NetworkManagerSecurityAdminRuleCollection/NetworkManagerSecurityAdminRule/NewAzNetworkManagerSecurityAdminRuleCommand.cs

```diff
@@ -90,20 +103,23 @@ public class NewAzNetworkManagerSecurityAdminRuleCommand : NetworkManagerSecurit
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Protocol of Rule. Valid values include 'Tcp', 'Udp', 'Icmp', 'Esp', 'Any', and 'Ah'.",
             ParameterSetName = "Custom")]
+        [ValidateSet("Tcp","Udp","Icmp","Esp","Any","Ah")]
         public string Protocol { get; set; }
 
         [Parameter(
             Mandatory = true,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Direction of Rule. Valid values include 'Inbound' and 'Outbound'.",
             ParameterSetName = "Custom")]
+        [ValidateSet("Inbound","Outbound")]
```

Comment:
Is it case sensitive or not?

Reply:
Good question, I did not take time to test each enum to see how we parse. I'll try and find time, if not we can always change to case insensitive in the next version.